sdk/confidentialledger/azure-security-confidentialledger/swagger/README.md

```diff
@@ -1,17 +1,39 @@
 ## Generate autorest code
-``` yaml
+``` yaml $(tag) == 'confidential-ledger'
 input-file: 
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/common.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/confidentialledger.json
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/0.1-preview/identityservice.json
+- https://github.com/Azure/azure-rest-api-specs/blob/main/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/common.json
```

Comment:
Another question is whether they can be in different version?

For now I see both is `2022-05-13`. Making 2 codegen pass make 2 version file `ConfidentialLedgerServiceVersion` and `ConfidentialLedgerIdentityServiceVersion`.
Is this necessary?

That probably a question for SDK arch board review. If you already get greenlight on it, ignore me.

Reply:
We need to generate the LedgerClient and IdentityClient separately because of a Swagger change we made. Since we removed operation status tags from Swagger, this is the only way we can do it. Christopher is doing it this way for dotnet, too.